src/ApiClient/PayPlugApiClient.php

```diff
@@ -55,9 +65,15 @@ public function initialise(string $secretKey): void
         );
     }
 
-    public function getAccount(): array
+    public function getAccount(bool $refresh = false): array
     {
-        return Authentication::getAccount($this->configuration)['httpResponse'] ?? [];
+        if ($refresh) {
+            $this->cache->delete($this->factoryName . '_current_account');
```

Comment:
```suggestion
            $this->cache->delete('payplug_account_' . substr($this->configuration['secretKey'], -4, 4));
```

On a pas besoin du nom de la factory car les deux ont les mêmes infos.
Par contre si tu configures deux comptes Payplug sur la même instance Sylius, il faut que les données soient différentes.